loadtest/control/simulcontroller/state.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package simulcontroller
+
+import (
+	"sync"
+)
+
+type ThreadInfo struct {
+	Id        string
+	ChannelId string
+	TeamId    string
+}
+
+type state struct {
```

Comment:
Each user has a separate store. I need a way for all users to share some data - in this case, common threads that they should all post replies to, so that we can get truly big threads.  

I noticed this pattern in the generative controller so I replicated here. Do you have something else in mind to share data across users?

Reply:
I see, we are trying to force a common behaviour. From my experience load-testing I've never thought that creating huge threads was ever an issue, I think it happened naturally as the result of replying to recent posts/replies. 

Either way, I am thinking we could still write some heuristic based on what we have in the store (maybe leveraging `ReplyCount`?) to achieve a similar behaviour without having direct communication between users. Overall I feel it's a bit of an anti-pattern which in the case of the generative controller is accepted cause its purpose is to purely create some data but here it doesn't look great as it further complicates an already elaborate logic.

Reply:
@streamer45 
How about I move this logic to the generative controller? That way people wanting to test loads for long running threads can first run the generative controller to create long threads, and the logic in the simulative controller remains mostly unchanged.

Reply:
That works as well. We can always come back to this later on if needed.